package.json

```diff
@@ -18,7 +18,7 @@
     "pretest": "npm run lint",
     "test": "lerna run --stream --concurrency=1 test",
     "test:max-file-size": "lerna run --stream test:max-file-size",
-    "lint": "lerna run --parallel --stream lint -- --silent",
+    "lint": "lerna run --parallel --stream lint",
```

Comment:
I don't believe this was working in the past... upgrading lerna made this work and caused some problems because pwa-kit-dev doesn't take unknown options like `silent`.

I don't want to touch the pwa-kit-dev commands in this PR. So i'm removing the silent option as I don't think it's super useful.